core/util/src/mill/util/Jvm.scala

```diff
@@ -725,25 +725,11 @@ object Jvm {
   }
 
   // Parse a list of repositories from their string representation
-  private[mill] def repoFromString(str: String, origin: String): Result[Seq[Repository]] = {
-    val spaceSep = "\\s+".r
-
-    val repoList =
-      if (spaceSep.findFirstIn(str).isEmpty)
-        str
-          .split('|')
-          .toSeq
-          .filter(_.nonEmpty)
-      else
-        spaceSep
-          .split(str)
-          .toSeq
-          .filter(_.nonEmpty)
-
+  private[mill] def reposFromStrings(repoList: Seq[String]): Result[Seq[Repository]] = {
     RepositoryParser.repositories(repoList).either match {
```

Comment:
@alexarchambault is the syntax supported by `RepositoryParser.repositories` documented anywhere? I'd like to link it in the documentation for the `//| repositories:` configuration key